ports/esp32/network_lan.c

```diff
@@ -232,6 +235,13 @@ static mp_obj_t get_lan(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_ar
             self->phy = esp_eth_phy_new_ksz80xx(&phy_config);
             break;
         #endif
+        #if CONFIG_ETH_USE_OPENETH
```

Comment:
Please split this addition of `PYH_OPENETH` into a separate commit within this PR.

Reply:
Will do